astroquery/exoplanet_archive/exoplanet_archive.py

```diff
@@ -0,0 +1,104 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+import json
+import os
+from astropy.utils.data import download_file
+from astropy.io import ascii
+import astropy.units as u
+
+__all__ = ['ExoplanetArchive']
+
+EXOPLANETS_CSV_URL = ('http://exoplanetarchive.ipac.caltech.edu/cgi-bin/'
+                      'nstedAPI/nph-nstedAPI?table=exoplanets')
+TIME_ATTRS = {'rowupdate': 'iso'}
+BOOL_ATTRS = ('pl_kepflag', 'pl_ttvflag', 'pl_k2flag', 'st_massblend',
+              'st_optmagblend', 'st_radblend', 'st_teffblend')
+
+
+class ExoplanetArchiveClass(object):
```

Comment:
Let's use `NasaExoplanetArchive`, which is the official full name of the archive and will avoid confusion with other archives?

Reply:
👍